doc/samples/SqlClientEventSource.cs

```diff
@@ -0,0 +1,57 @@
+﻿// <Snippet1>
+using System;
+using System.Diagnostics.Tracing;
+using Microsoft.Data.SqlClient;
+
+// This listener class will listen for events from the SqlClientEventSource class.
+// SqlClientEventSource is an implementation of the EventSource class which gives 
+// it the ability to create events for Event Tracing for Windows (ETW).
+public class SqlClientListener : EventListener
+{
+    protected override void OnEventSourceCreated(EventSource eventSource)
+    {
+        // Only enable ETW events from SqlClientEventSource.
+        if (eventSource.Name.Equals("Microsoft.Data.SqlClient.EventSource"))
+        {
+            // Capture basic application flow events.
+            EnableEvents(eventSource, EventLevel.Informational, (EventKeywords)2);
```

Comment:
Can you explain the reason for using the number 2?

Reply:
Updated the comment. This sample will also be accompanied by the table containing all the event keywords.